src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -661,4 +661,11 @@
   <data name="SqlQueryProcessorRanOutOfInternalResourcesException" xml:space="preserve">
     <value>The query processor ran out of internal resources and could not produce a query plan. Please limit your parameters and retry.</value>
   </data>
-</root>
+  <data name="ResourceNotAllowedRestrictedByClinicalScopes" xml:space="preserve">
+    <value>Resource type {0} is not allowed, additional clinical scopes should be requested to allow searching using this type.</value>
```

Comment:
Nit: Change to `Resource type {0} is not allowed. Additional clinical scopes should be requested to allow searching using this type.`, to use the same pattern as the error message below?

Reply:
Updated.